app/templates/administrative-units/administrative-unit/core-data/edit.hbs

```diff
@@ -13,284 +13,224 @@
-                    <div class="au-o-grid__item au-u-3-4">
-                      <AuControlRadio
-                        @label="Ja"
-                        @name="is-cross-border"
-                        @value={{@model.administrativeUnit.crossBorder}}
-                        @identifier="cross-border-true"
-                        @onChange={{fn
-                          (mut @model.administrativeUnit.crossBorder)
-                          true
-                        }}
-                        checked={{@model.administrativeUnit.crossBorder}}
-                      />
+        <EditCard>
+          <:title>Bestuur van de eredienst</:title>
+          <:card as |Grid|>
+            <Grid as |Item|>
+              <Item @labelFor="worship-service-name">
+                <:label>Naam eredienst</:label>
+                <:content>
+                  <AuInput
+                    @width="block"
+                    @value={{@model.administrativeUnit.name}}
+                    id="worship-service-name"
+                  />
+                </:content>
+              </Item>
+              <Item @labelFor="status-select">
+                <:label>Status</:label>
+                <:content>
+                  <PowerSelect
+                    @noMatchesMessage="Geen resultaten"
+                    @options={{@model.statuses}}
+                    @selected={{@model.administrativeUnit.organizationStatus}}
+                    @onChange={{fn
+                      (mut @model.administrativeUnit.organizationStatus)
+                    }}
+                    @disabled={{true}}
+                    @triggerId="status-select"
+                    as |status|
+                  >
+                    {{status.label}}
+                  </PowerSelect>
+                </:content>
+              </Item>
+              <Item @labelFor="recognized-worship-type-select">
+                <:label>Type eredienst</:label>
+                <:content>
+                  <RecognizedWorshipTypeSelect
+                    @selected={{this.model.administrativeUnit.recognizedWorshipType}}
+                    @onChange={{fn
+                      (mut this.administrativeUnit.recognizedWorshipType)
+                    }}
+                  />
+                </:content>
+              </Item>
+            </Grid>
+            {{#if (eq @model.administrativeUnit.constructor.modelName "worship-service")}}
+              <Grid as |Item|>
+                <Item @labelFor="denomination">
+                  <:label>Strekking</:label>
+                  <:content>
+                    <AuInput
+                      @width="block"
+                      @value={{@model.administrativeUnit.denomination}}
+                      id="denomination"
+                    />
+                  </:content>
+                </Item>
+              </Grid>
+            {{/if}}
+            {{#if (eq @model.administrativeUnit.constructor.modelName "worship-service")}}
```

Comment:
This seems to be the same check as the if on line 60, so I think that can be combined into 1?